apps/redi-connect/src/pages/app/profile/Profile.generated.ts

```diff
@@ -1,36 +1,82 @@
+}>
 
+export type ProfilePageQueryQuery = {
+  __typename?: 'Query'
+  conProfile: {
+    __typename?: 'ConProfile'
+    userId: string
+    age?: number | null
+    birthDate?: any | null
+    categories: Array<Types.MentoringTopic>
+    createdAt: any
+    doesNotHaveAvailableMentorshipSlot: boolean
+    email: string
+    expectations?: string | null
+    firstName: string
+    fullName: string
+    gender?: Types.Gender | null
+    githubProfileUrl?: string | null
+    id: string
+    languages?: Array<Types.Language> | null
+    lastName: string
+    linkedInProfileUrl?: string | null
+    loopbackUserId: string
+    menteeCountCapacity?: number | null
+    mentee_highestEducationLevel?: Types.EducationLevel | null
+    mentee_occupationCategoryId?: Types.OccupationCategory | null
+    mentee_occupationJob_placeOfEmployment?: string | null
+    mentee_occupationJob_position?: string | null
+    mentee_occupationLookingForJob_what?: string | null
+    mentee_occupationOther_description?: string | null
+    mentee_occupationStudent_studyName?: string | null
+    mentee_occupationStudent_studyPlace?: string | null
+    mentor_isPartnershipMentor?: boolean | null
+    mentor_occupation?: string | null
+    mentor_workPlace?: string | null
+    optOutOfMenteesFromOtherRediLocation: boolean
+    personalDescription?: string | null
+    profileAvatarImageS3Key?: string | null
+    profileStatus: Types.ConnectProfileStatus
+    rediLocation: Types.RediLocation
+    slackUsername?: string | null
+    telephoneNumber?: string | null
+    updatedAt: any
+    userActivatedAt?: any | null
+    userType: Types.UserType
+  }
+}
 
 export const ProfilePageQueryDocument = `
     query ProfilePageQuery($id: ID!) {
   conProfile(id: $id) {
     ...AllConProfileFields
   }
 }
-    ${AllConProfileFieldsFragmentDoc}`;
+    ${AllConProfileFieldsFragmentDoc}`
 export const useProfilePageQueryQuery = <
-      TData = ProfilePageQueryQuery,
-      TError = unknown
-    >(
-      variables: ProfilePageQueryQueryVariables,
-      options?: UseQueryOptions<ProfilePageQueryQuery, TError, TData>
-    ) =>
-    useQuery<ProfilePageQueryQuery, TError, TData>(
-      ['ProfilePageQuery', variables],
-      fetcher<ProfilePageQueryQuery, ProfilePageQueryQueryVariables>(ProfilePageQueryDocument, variables),
-      options
-    );
+  TData = ProfilePageQueryQuery,
+  TError = unknown
+>(
+  variables: ProfilePageQueryQueryVariables,
+  options?: UseQueryOptions<ProfilePageQueryQuery, TError, TData>
+) =>
+  useQuery<ProfilePageQueryQuery, TError, TData>(
+    ['ProfilePageQuery', variables],
+    fetcher<ProfilePageQueryQuery, ProfilePageQueryQueryVariables>(
+      ProfilePageQueryDocument,
+      variables
+    ),
+    options
+  )
 
-useProfilePageQueryQuery.getKey = (variables: ProfilePageQueryQueryVariables) => ['ProfilePageQuery', variables];
-;
+useProfilePageQueryQuery.getKey = (
+  variables: ProfilePageQueryQueryVariables
+) => ['ProfilePageQuery', variables]
```

Comment:
This file is an auto-generated file. I guess you saved this file when it was open and it was auto-styled without any code change. Am I right? If so, can we take this file back to it's original version?